.readme-partials.yml

```diff
@@ -0,0 +1,7 @@
+body: |-
+        ### Troubleshooting
+        #### Emulator returning `DEADLINE_EXCEEDED`, `java.lang.OutOfMemoryError`
+        *Reference Issue: https://github.com/googleapis/nodejs-datastore/issues/95*
+        When using the emulator, you may experience errors such as "DEADLINE_EXCEEDED" within your application, corresponding to an error in the emulator: "java.lang.OutOfMemoryError". These errors are unique to the emulator environment and will not persist in production.
+        
+        A workaround is available, provided by [@ohmpatel1997](https://github.com/ohmpatel1997) [here](https://github.com/googleapis/nodejs-datastore/issues/95#issuecomment-554387312).
```

Comment:
I'd be tempted to include the workaround here, explaining that you simply run Node.js with more memory; but with a link to the issue at the end of the line?

Reply:
I was tempted as well, although I didn't actually understand how to pass those arguments to the emulator. I figured the comment would make sense to emulator users, so I just linked. But if you know the full command to pass those, I will pop it in here!